internal/server/data/migrations.go

```diff
@@ -185,14 +194,17 @@ func setDestinationLastSeenAt() *migrator.Migration {
 	return &migrator.Migration{
 		ID: "202207041724",
 		Migrate: func(tx *gorm.DB) error {
-			if tx.Migrator().HasColumn(&models.Destination{}, "last_seen_at") {
+			if migrator.HasColumn(tx, "destinations", "last_seen_at") {
 				return nil
 			}
 
-			if err := tx.Migrator().AddColumn(&models.Destination{}, "last_seen_at"); err != nil {
+			stmt := `ALTER TABLE destinations ADD COLUMN last_seen_at timestamp with time zone`
```

Comment:
This is good, I prefer the timezone in the db, but do you know if other tables in postgres have been getting `timestamp with time zone`? might cause subtle problems somewhere if they're different.

Reply:
Ya, this matches what `gorm.AutoMigrate` produces, you can see that here: https://github.com/infrahq/infra/pull/2843/files#r940287138